documentation/asciidoc/topics/proc_configuring_xsite_within_clusters.adoc

```diff
@@ -9,20 +9,13 @@ For evaluation and demonstration purposes, you can configure {brandname} to back
 . Create an `Infinispan` CR for each {brandname} cluster.
 . Specify the name of the local site with `spec.service.sites.local.name`.
 . Set `ClusterIP` as the value of the `spec.service.sites.local.expose.type` field.
-. Provide the {brandname} cluster hostname as the URL for each backup location with `spec.service.sites.locations`.
-+
-The following is an example `Infinispan` CR:
+. Provide the name of the {brandname} cluster that acts as a backup location with `spec.service.sites.locations.clusterName`.
```

Comment:
Optionally namespace can be configured for utilizing the same cluster name internally